app/graphql/types/list_albums.rb

```diff
@@ -0,0 +1,55 @@
+require 'search_object/plugin/graphql'
+
+module Types
+  class ListAlbums < QueryType
+    include SearchObject.module(:graphql)
```

Comment:
This should be in a resolver instead doing it in a type, to be a little more clean and don't have everything in only one file:

https://www.howtographql.com/graphql-ruby/7-filtering/